### PR TITLE
Fixing the init bug

### PR DESCRIPTION
**Pull request by dushr**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -55,14 +58,12 @@ FlavourTheserous.prototype.initFirebase = function() {
...
-    // Get a reference to the database service
-    var database = firebase.database();
 
     var tags = ["Meaty", "Roasted", "Floral Fruit", "Berry and Bush", "Citrusy", "Creamy Fruity", "Fresh Fruity", "Woodland", "Spicy", "Green and Grassy", "Brine and Salt", "Marine", "Sulfurous", "Musterdy", "Earthy", "Cheesy"];
     
-    for (x in tags) {
+    for (var x in tags) {
         console.log(x);
-        savedata("tags", x);
+        saveData("tags", x);
```
Comment (vyasakanksha) [inline main.js:66]: You seem to have python on your mind. This saves integers and not the tags.

File: main.js [hunk 2]
```diff
@@ -2,18 +2,21 @@
...
-    ref = database.ref(db);
+    var database = firebase.database();
+    var ref = database.ref(db);
     return ref.getById(id)
 }
 
 // Initializes FriendlyChat.
 function FlavourTheserous() {
-  this.checkSetup();
+  //this.checkSetup();
```
Comment (vyasakanksha) [inline main.js:19]: Do you want to get rid of this forever, or just for now?

**Review by vyasakanksha — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (dushr) [reply main.js:66]: True, it should be 
```javascript
tags.forEach(x => saveData("tags", x))
```

File: main.js (hunk 2 above)
Comment (dushr) [reply main.js:19]: I don't know; I wasn't working on this version. I'd been working on the react version

https://github.com/dushr/thesilkroute/tree/react

**Pull request merged**